.gitignore

```diff
@@ -1,4 +1,5 @@
 __pycache__
 .DS_Store
 env/
-.vscode/
\ No newline at end of file
+.vscode/
+commands/twitter_token.py
```

Comment:
Can we remove this redundant file (twitter_token.py)? It's too specific to be put in a .gitignore anyway.

Reply:
Done.